packages/preview/famnit/0.1.0/template/main.typ

```diff
@@ -0,0 +1,49 @@
+#import "@preview/famnit:0.1.0": project
+
+#show project.with(
```

Comment:
There are a few syntax errors in this file.

Reply:
Forgot the commas :sweat_smile: . I fixed them